### PR TITLE
Feat: Ensure single axon and remove dummy dendrites

### PR DESCRIPTION
**Pull request by arnaudon**

### CODE REVIEW

File: .pylintrc [hunk 1]
```diff
@@ -14,7 +14,7 @@ max-line-length=100
 
 [DESIGN]
 # Maximum number of arguments for function / method
-max-args=8
+max-args=10
```
[the remark this answers is not in the document]
Comment (arnaudon) [reply .pylintrc]: this has no effect on ruff thingy it seems, I revert

File: src/morphology_workflows/curation.py [hunk 2]
```diff
@@ -425,21 +425,66 @@ def fix_root_section(morph, min_length=_ZERO_LENGTH):
...
+
+def _ensure_axon_at_soma(morph):
+    """Ensure axon is connected to soma."""
+    for root_section in morph.root_sections:
+        root_section_type = root_section.type
+        for section in root_section.iter():
+            if section.type != root_section_type:
+                morph.append_root_section(section)
+                morph.delete_section(section, recursive=True)
+                break
```
[the remark this answers is not in the document]
Comment (adrien-berchet) [reply src/morphology_workflows/curation.py:447]: It's very unlikely indeed :)

[the remark this answers is not in the document]
Comment (adrien-berchet) [reply src/morphology_workflows/curation.py:447]: (well, I hope at least!)

File: tox.ini [hunk 3]
```diff
@@ -62,7 +62,7 @@ commands =
 basepython = python3.8
 
 [testenv:lint]
-basepython = python3.8
+basepython = python3
```
Comment (adrien-berchet) [inline tox.ini:65]: You will never install Py38, will you? :smile:

Comment (arnaudon) [reply tox.ini:65]: I use bb5 python in latest modules

Comment (arnaudon) [inline tox.ini:65]:
```suggestion
basepython = python3.8
```

File: .pre-commit-config.yaml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 default_language_version:
-  python: python3.8
+  python: python3
```
Comment (arnaudon) [inline .pre-commit-config.yaml:2]:
```suggestion
  python: python3.8
```

**Pull request merged**